### DISCUSSION TITLE
repository-url not set when using vercel deploy button with redirect

### DISCUSSION THREAD

**Discussion by neokry** [vercel/community#1421] [Help]
### Summary

I'm creating a simple website builder that utilizes vercel for hosting. After users deploy their sites I need the github repository url but when redirecting that query param is never set. This issue is happening with public and private github repositories I'm creating through vercel. Is this param only set conditionally?

Below is the deploy URL I'm using to test this out:

https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fneokry%2Fnoun-site&env=NEXT_PUBLIC_ALCHEMY_KEY,NEXT_PUBLIC_TOKEN_CONTRACT&redirect-url=http%3A%2F%2F127.0.0.1%3A3000%2F

And here is the query paramters I recieve back (repository-url is blank):

http://127.0.0.1:3000/?project-dashboard-url=https%3A%2F%2Fvercel.com%2Fartiva%2Fundefined&deployment-dashboard-url=https%3A%2F%2Fvercel.com%2Fartiva%2Fnouns-test-3%2FBdkmgpyxqk7z3Mf1omDUHL9Q5D1M&deployment-url=https%3A%2F%2Fnouns-test-3-pya3nvhap-neokry.vercel.app&repository-url=

**Comment by amyegan** [comment]
Is it giving you the same result for other repos? For example, the [Next.js Boilerplate template with a `localhost:3000` redirect](https://vercel.com/new/clone/hidden?demo-title=Next.js+Boilerplate&demo-description=A+Next.js+app+and+a+Serverless+Function+API.&demo-url=https://nextjs-template.vercel.app/&demo-image=//images.ctfassets.net/e5382hct74si/29l0UrpsQB6uDlmShjiJ72/f44f1f951062bcefa9959e1985cb61af/LightMode_CreateNextApp.png&project-name=Next.js+Boilerplate&repository-name=nextjs-boilerplate&repository-url=https://github.com/vercel/vercel/tree/main/examples/nextjs&from=templates&redirect-url=http://127.0.0.1:3000&deploymentIds=dpl_BeBeF3xsoZccZYsjT75Rj6UV967W)
